src/components/atoms/toaster/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect, useRef, useState } from "react"
+import {
+  AppearanceTypes,
+  Placement,
+  ToastProps,
+} from "react-toast-notifications"
+import AlertIcon from "../../fundamentals/icons/alert-icon"
+import CheckCircleIcon from "../../fundamentals/icons/check-circle-icon"
+import CrossIcon from "../../fundamentals/icons/cross-icon"
+import InfoIcon from "../../fundamentals/icons/info-icon"
+import XCircleIcon from "../../fundamentals/icons/x-circle-icon"
+
+type ToasterContentProps = {
+  message: string
+  title: string
+}
+
+const ICON_SIZE = 20
+
+const toastStates = (placement: Placement) => ({
+  entering: { transform: getTranslate(placement) },
+  entered: { transform: "translate3d(0,0,0)" },
+  exiting: { transform: "scale(0.66)", opacity: 0 },
+  exited: { transform: "scale(0.66)", opacity: 0 },
+})
+
+const Container: React.FC<ToastProps> = ({
+  children,
+  transitionDuration,
+  transitionState,
+  appearance,
+  placement,
+  onDismiss,
+}) => {
+  const [height, setHeight] = useState<string | number>("auto")
+  const elementRef = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    if (transitionState === "entered") {
+      const el = elementRef.current
+      if (el) {
+        setHeight(el.offsetHeight + 8)
+      }
+    }
+    if (transitionState === "exiting") {
+      setHeight(0)
+    }
+  }, [transitionState])
+
+  return (
+    <div
+      ref={elementRef}
+      style={{
+        height,
+        transition: `height ${transitionDuration - 100}ms 100ms`,
+      }}
+    >
+      <div
+        className="flex items-start bg-grey-0 p-base border border-grey-20 rounded-rounded shadow-toaster w-[380px] mb-xsmall last:mb-0"
+        style={{
+          transitionDuration: `${transitionDuration}ms`,
+          transition: `transform ${transitionDuration}ms cubic-bezier(0.2, 0, 0, 1), opacity ${transitionDuration}ms`,
+          zIndex: 1000,
+          ...toastStates(placement)[transitionState],
+        }}
+      >
+        <div>{getIcon(appearance)}</div>
+        <div className="flex flex-col ml-small mr-base gap-y-2xsmall flex-grow">
+          {children}
+        </div>
+        <div>
+          {/* @ts-ignore:next-line */}
+          <div role="button" onClick={onDismiss} className="cursor-pointer">
+            <CrossIcon size={ICON_SIZE} className="text-grey-40" />
+          </div>
```

Comment:
just curious to know why a `div` instead of `button`? Tried using a button and couldn't notice any differences (visually), but might have missed it :smile: